core/primitives/src/sharding.rs

```diff
@@ -439,12 +468,22 @@ impl ShardChunkHeader {
         }
     }
 
+    #[inline]
+    pub fn bandwidth_requests(&self) -> Option<&BandwidthRequests> {
+        match self {
+            ShardChunkHeader::V1(_) | ShardChunkHeader::V2(_) => None,
+            ShardChunkHeader::V3(header) => header.inner.bandwidth_requests(),
+        }
+    }
+
     /// Returns whether the header is valid for given `ProtocolVersion`.
     pub fn valid_for(&self, version: ProtocolVersion) -> bool {
```

Comment:
Reminder to myself - make sure that we check this in witness validation. Didn't matter before, but might matter now.